assets/src/state.ts

```diff
@@ -86,6 +86,9 @@ export interface State {
   selectedNotification?: Notification
   openView: OpenView
   previousView: OpenView
+  swingsViewScrollPosition: number
+  showPastSwings: boolean
```

Comment:
I considered having a single mapping of views to scroll positions, but sadly that left the swings view show / hide past swings state as sort of the odd one out. I like the idea of encapsulating the view state somehow but basically wasn't able to find a way of organizing it that I liked. If we add more views that have similar state remembering logic it's definitely something I'll keep in mind, though.

Reply:
Yeah, agree that this is something that may be worth considering down the line but isn't pressing now! One thought is maybe there is a core view state type like 
```
interface ViewState {
scrollPosition: number
}
``` and each view can extend that type with its own relevant data 
```
interface wingsViewState extends ViewState {
showPastSwings: boolean
}
``` That may create more trouble that it is worth at this point though! 

Thinking of a core view state reminded me that there is also the `LateView` which as I understand this PR will not be saving scroll position. Based on the scope of the ticket that seems expected, but wanted to mention it just in case.

Reply:
I like that general idea! I think for now I'll hold off on actually implementing it but that sounds like it's probably the right approach. And you make a good point re: late view, that is out of scope for this particular ticket but presumably we'll want to revisit it as well at some point.